src/payment/strategies/klarna/klarna-payment-strategy.ts

```diff
@@ -130,6 +131,10 @@ export default class KlarnaPaymentStrategy implements PaymentStrategy {
         return data;
     }
 
+    private _needsStateCode(countryCode: string) {
+        return includes(this._supportedOCCountries, countryCode);
```

Comment:
do you need state code for NZ? I believe NZ doesn't have state codes.

Also, it might make more sense to have two different arrays to better express the intent of them
```js
private _countriesRequiringStates = ['AU', 'NZ'];
private _supportedCountries = ['AT', 'DE', 'DK', 'FI', 'GB', 'NL', 'NO', 'SE', 'CH', ...this._countriesRequiringStates];
```

Reply:
yes, It's required by Klarna.

Reply:
As discussed offline, I don't think NZ will return any stateOrProvinceCode, as it's not a dropdown in our UI.

<img width="585" alt="Screen Shot 2020-04-02 at 9 49 46 am" src="https://user-images.githubusercontent.com/1621894/78196801-371e3400-74cf-11ea-814e-2a135dff034b.png">

Reply:
That's right I'm going to apply this change, thanks for the explanation